daemon/api_buy_unsupp.go

```diff
@@ -75,48 +75,26 @@ func readyToBuy(c *Command, r *http.Request, user *auth.UserState) Response {
 }
 
 func convertBuyError(err error) Response {
```

Comment:
Would it make sense to use `*apiError` as  the return type here? It looks like that would match all the return statements in the body.

If you make this change, it might also be worth changing the function's callers use `rspe` as the name of the variable holding the return value.

Reply:
this code will be removed at some point (hopefully sooner rather than later, the store dropped the unused support for this recently as well), that's why the file has the "unsupp" suffix, so I don't plan to change it more than minimally to keep it compiling and tests happy. I had to touch it because `resp` is going away.

Reply:
Fair enough.